### PR TITLE
PALS-296: Removed survey

### PR DESCRIPTION
**Pull request by robert-hirsch-po**
Resolves PALS-296

## Description

Removed survey script and added error handling

## Checks

**Have you written any necessary unit tests?**

- [ ] Yes
- [ ] Not yet
- [x] Not applicable

**Have you added or updated any necessary integration tests?**

- [ ] Yes
- [ ] Not yet
- [x] Not applicable

**Browsers tested on:**

- [x] Chrome
  - [ ] Mobile
- [ ] Safari
  - [ ] Mobile
- [ ] Firefox
  - [ ] Mobile
- [ ] Edge
- [ ] IE11
- [ ] Not applicable

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -271,29 +271,28 @@ export class AppComponent {
 
   ngOnInit() {
     if (isPlatformBrowser(this.platformId)) {
-      this._script.loadScript('ethnio-survey');
```
Comment (robert-hirsch-po) [inline src/app/app.component.ts]: Look here

File: src/app/_services/script.store.ts [hunk 2]
```diff
@@ -6,8 +6,6 @@ interface Scripts {
 export const ScriptStore: Scripts[] = [
   // Zendesk chat widget
   { name: 'zendesk', src: '../../assets/js/zendesk.js' },
-  // Ethnio, current survey: Annual demographic survey on image groups, search results, asset page
```
Comment (robert-hirsch-po) [inline src/app/_services/script.store.ts]: Look here

File: src/app/_services/script.service.ts [hunk 3]
```diff
@@ -34,21 +34,27 @@ export class ScriptService {
 
     loadScript(name: string) {
         return new Promise((resolve, reject) => {
+            if (!this.scripts[name]) {
```
Comment (robert-hirsch-po) [inline src/app/_services/script.service.ts]: Look at this block for error handling

**Comment by eslawski** [comment]
I looks like there might be some linting differences that are sneaking in here?

**Pull request merged**